test/util/index.js

```diff
@@ -28,3 +28,11 @@ export function createMap(t, options, callback) {
 
     return map;
 }
+
+export function equalWithPrecision(test, expected, actual, multiplier, message, extra) {
```

Comment:
What's the rationale for moving `equalWithPrecision` to a standalone function? Since this is wrapping tap's built-in `equals` assertion, to me it makes sense to leave it as a custom assertion method called like all other tap assertions, instead of switching to this functional API - unless we have a strong reason that I'm missing.

Reply:
My main motivation is moving away from Tap-specific things — e.g. I might try switching to `tape` in future to fix conflicting deps like `esm`, and `tape` is much more minimalistic in design and encourages making all helpers explicit rather than augmenting the prototype. And if we ever consider switching to a different testing ecosystem (e.g. to Jest), this will make it easier.

Reply:
OK, fair enough - I personally would wait until we actually _do_ make the change away from tap, but if you think that'll be coming down the pike soon then sure. No biggie either way, anyway :smile: